src/api/types.ts

```diff
@@ -207,7 +210,8 @@ export type RenderType =
   | 'textarea'
   | 'toggle'
   | 'fixed-value'
-  | 'file';
+  | 'file'
+  | 'testOrder';
```

Comment:
Do you need this since we are planning on reusing the `select` rendering?

Reply:
We need this because we need an orders handler that will be handling order related functions

Reply:
The handler isn't mapped by the `rendering` prop but rather `type`.